src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -1682,27 +1704,4 @@ private CpuClass() {
             return line -> line.replaceFirst("[^:]*+: ", "");
         }
     }
-
-    /**
-     * Was assertion enabled for the Jvm class when it was initialised.
-     *
-     * @return if assertions were enabled.
-     */
-    public static boolean isAssertEnabled() {
-        return ASSERT_ENABLED;
-    }
-
-    public static boolean supportThread() {
```

Comment:
possibly a breaking API change!

Reply:
This static method was moved to before the non static members by the IDE